test/integration/features/basic.feature

```diff
@@ -10,6 +10,42 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
+  Scenario: OpenShift is ready after startup
+    After startup of Minishift OpenShift instance should respond correctly on its html endpoints
+    and OpenShift web console should be accessible.
+    Given Minishift has state "Running"
+     Then body of HTTP request to "OpenShift" at "/healthz" contains "ok"
```

Comment:
@hferentschik Yes, that is correct, it accepts both dynamic and explicit addresses.

> Technically though, I think we should be looking at the returned status code

To be honest, I am not sure here. Spent some time on going through documentation and I have not found an information about these endpoints - does they return only "ok" response, or do they have more responses - eg. "not ready"?

However I am going to add check for response code into implementation, so we can later switch from one to second just by altering steps in the .feature file.